django_guid/log_filters.py

```diff
@@ -14,5 +14,5 @@ def filter(self, record: LogRecord) -> bool:
         :param record: Log record
         :return: True
         """
-        record.correlation_id = GuidMiddleware.get_guid()
+        record.__setattr__('correlation_id', GuidMiddleware.get_guid())
```

Comment:
I’m not sure why this change was made? What’s the benefit of using setattr here?  
If I’m missing something and it’s better I’d like it to be written like this instead:
```python
setattr(record, 'correlation_id', GuidMiddleware.get_guid())
```

Reply:
Yeah agree we shouldn't use hidden methods. The original implementation is probably fine - mypy was just complaining, so I temporarily tried the suggested implementation and forgot to revert. I'll push a fix later.

Reply:
Hmm, I'm no MyPy expert, but it's just a string, so I guess you can declare it as such?

Reply:
Reverted to what it was :)